jhbuild/modtypes/__init__.py

```diff
@@ -345,6 +346,129 @@ def _strip_debug_symbols(self, destdir_prefix, installroot):
             except Exception as e:
                 pass
 
+    def _find_exec(self, destdir_prefix):
+        exec_files = []
+        # filter out files to strip
+        for filename in fileutils.accumulate_dirtree_contents(destdir_prefix):
```

Comment:
Can you fix this loop to be similar to the _strip_debug_symbol one? You do not need the hacks here. They are added for the purpose of stripping symbols. Someone must have not realized that the function is also used for ldd.